types/test/plugin-test.ts

```diff
@@ -15,6 +15,9 @@ const plugin: PluginObject<Option> = {
 }
 const installer: PluginFunction<Option> = function(Vue, option) { }
 
+Vue.use(plugin);
```

Comment:
this one should yield an error though 🤔 

The tests that must be added are:

```ts
function NoOptions(_Vue: Vue): void {}
function OptionalOption(_Vue: Vue, options?: Option): void {}

Vue.use(NoOptions)
Vue.use(OptionalOption)
Vue.use(OptionalOption, { prefix: '', suffix: '' })
```

I'm realizing that the types you put in vue.d.ts are different from what you tried at https://www.typescriptlang.org/play/index.html#code/KYDwDg9gTgLgBDAnmYcAKAbArgcwJYB2AYlgQMYx4QEA8AKgHxwC8cAFAG5bABccAatwA0cCGErUAzgH4+dAJQsmHCHgAmAbgBQWwjGBQAZgEMyqALLBJk4zmAAlYJDgBvLXA9w7MNnAC2VjZ2ANLAiHySMFCEOHDyEVEx2gC+OnoGJmZwlta2Vpi4hADy4lQEkq5aAJABuXaSfDlBDk4QKTpIKNmBeZIF+MSkFGUs7HBcvALC-j12jbP52AMlEuVx2oZDq911i4UEbO6eE3yCwELVYqsNO819S8WlUiJa8eOqapWpWpvk2wByEBWZUkbAA+mdTtw3ip1K5Ur9htQ4MDqMYMKiDhDuFDzqInuVZLdev1HtcYR94WkCPojKZUGdKp5PFhJMB6Aw2GAHgQ+KTBn8yhyRFcQUSFHxYZojszWcBfNz9nyeSRBdQaKQANYECAAdwIDBEADoTaKpHwtTr9QBtAC6cUlH20zI8coVPPmu3u+1VSIIIoJnru-MxFQp6naOjIUngE1GLmScGMFTO2mj5XgtWajmcrATSYqTTyObaOiL9X5vgmIhcMy9fCzxdacET8g0cAA9B2vBAIGotBMjW7iRWeTW63cGwsSy31p3uzhe-3B27AaH5APuEO2WxMejMRuVzu9xiCePG-Up7sZ8kN1ouwgABaoQwQDAYPUxOCSR8QLAYT4ACNUFAbk+1QQgn1QaM-DAPAMAMTdgG3eURz2AZ1iQlDfHLdDCBrVttCPVDcO9AZx0MQwr2zZtCKw4dSP5c8FhuAByV8IFY2c2yAA

Reply:
It passes also in current dev

Thanks for that advises.

Indeed is a minor difference to the mentioned playground. The reason is 
`Vue.use(installer, new Option, new Option, new Option);`
Would fail, since the definition hits in the master the `any` case of use, which would not work with `unknown`.  (I am even not sure if this was intended in the master)

Reply:
Please let me know, if the `any` fallback was intended on `Vue.use<PluginOption>`. This could behavior could be restored by a additional definition  ` use<T>(plugin: PluginObject<T> | PluginFunction<T>, option1: any, option2: any, ...options: any[] ): VueConstructor<V>;`. It looks perhaps a bit odd, but `use` and a given type should fallback at n+1( n>1 ) arguments to`any` according to master.
( Please see: https://tinyurl.com/y3lcapvv )